### ISSUE TITLE
A11y_code.visualstudio.com_DocsRemote_LinuxPrerequisites_K4W: <html> element must have a lang attribute 

### ISSUE THREAD

**Issue by Mohini78** [microsoft/vscode-docs#2789]
“[Check out Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
Environment Details:
URL: [https://code.visualstudio.com](https://code.visualstudio.com)
Windows Version: Windows10
Browser Version:
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
MAS Violated: MAS3.1.1
Other Information : AI Auto issue

Repro Steps:
Navigate to the url: [https://code.visualstudio.com/docs/remote/linux](https://code.visualstudio.com/docs/remote/linux)
follow below steps:
Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com)
Navigate to "Docs" link.
From left navigation, select "Remote".
Navigate to "Linux Prerequisites" link.
Navigate to:
![image](https://user-images.githubusercontent.com/51443731/59992913-b55dc800-966b-11e9-82be-5374997bee41.png)
Use Accessibility Insights​ tool and run it and observe the issues.

Actual:
<html> element do not have a lang attribute

Expected:
<html> element must have a lang attribute.
Add a lang attribute to the html element (e.g. <html lang="en">) whose value represents the primary language of document.
Make sure you identify a language in the opening <html> element and spell the attribute correctly.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

User Impact:
When configuring a screen reader, users select a default language. If the language of a webpage is not specified, the screen reader assumes the default language set by the user. Language settings become an issue for users who speak multiple languages and access website in more than one language. It is essential to specify a language and ensure that it is valid so website text is pronounced correctly.

MAS Reference:
MAS 3.1.1 -
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={eb8515c6-18d7-4af0-b47e-c46958b1b50e}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={eb8515c6-18d7-4af0-b47e-c46958b1b50e})

Attachment for Reference:
![A11y_code visualstudio com_DocsRemote_LinuxPrerequisites_K4W_1](https://user-images.githubusercontent.com/51443731/59994298-f3111f80-9670-11e9-846b-3af5589752ee.png)
![A11y_code visualstudio com_DocsRemote_LinuxPrerequisites_K4W_2](https://user-images.githubusercontent.com/51443731/59994299-f3111f80-9670-11e9-8c0b-f2d2256088dd.png)

**Comment by gregvanl** [comment]
Duplicate of https://github.com/microsoft/vscode-docs/issues/2617
